tests/integration/test_fim/test_files/test_report_changes/test_file_size_default.py

```diff
@@ -63,13 +63,13 @@
 import os
 
 import pytest
-from test_fim.test_files.test_report_changes.common import generate_string, translate_size, make_diff_file_path, \
-    disable_rt_delay, restore_rt_delay
+from test_fim.test_files.test_report_changes.common import generate_string, translate_size, make_diff_file_path
```

Comment:
If you see that the method is used outside of the report_changes test you should move it, or copy it to a place where will use it for the fim module and start using that. If not, the framework continues to get dirty.

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/3503/commits/7c5a46a36366ade14be090824ef103620354c2c5